tests/monitoring/monitoring.go

```diff
@@ -251,17 +254,39 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", func() {
 		checks.SkipIfPrometheusRuleIsNotEnabled(virtClient)
 	})
 
-	Context("Alerts runbooks", func() {
-		It("Should have available URLs", func() {
-			alerts, err := getAlerts(virtClient)
+	Context("Kubevirt alert rules", func() {
+		BeforeEach(func() {
+			monv1 = virtClient.PrometheusClient().MonitoringV1()
```

Comment:
@assafad I think it's better to create this variable in-place with `monv1 :=` instead re-assign it. Then you won't need to import the `kubevirt.io/client-go/generated/prometheus-operator/clientset/versioned/typed/monitoring/v1` package

Reply:
Done